app/models/health/member_status_report.rb

```diff
@@ -62,7 +63,7 @@ def run!
           cp_outreach_status: pr.outreach_status,
           cp_last_contact_date: most_recent_qualifying_activity&.date_of_activity,
           cp_last_contact_face: client_recent_face_to_face(most_recent_qualifying_activity),
-          cp_contact_face: any_face_to_face_for_patient_in_range(patient, report_range),
+          cp_contact_face: any_face_to_face_for_patient_in_range(patient),
```

Comment:
this looks like a bug (the change), unless report_range is defined somewhere as a method or attr_accessor.

Reply:
which it is.